### DISCUSSION TITLE
Joined Copilot Queue very long time ago

### DISCUSSION THREAD

**Discussion by skandabhairava** [community/community#6651] [Copilot Conversations] [answered]
I signed up for copilot a long time ago(4-5 months ago, almost a few days after release of copilot for preview) and I also selected the option that I code most of the time and I’ve still not received access to it. Many of my friends/colleagues already received copilot a few days from when they signed up. All my personal info is correct, my email info is correct as I receive notification from github sometimes. I haven’t received any email about copilot from github(even spam folder is empty). I can’t even reapply for copilot as it just asks me to wait for my turn. I also found out that copilot will become a commercial product after the preview phase ends, i would like to give copilot a try before it becomes paid.

Is there something I’m doing wrong??

**Comment by darkanubis0100** [comment, marked as answer]
We are in the same place and curiously with the same amount of time. The GitHub guys still have me on the waiting list and nothing to allow me access. I'm very excited to try it out.

**Reply by TiagoCerqueira** [reply]
Still waiting too with same amount of time. I don't understand why this is so long !

**Reply by thuanha-groove** [reply]
it's too long. I have been waiting for a couple of months

**Comment by linktlh** [comment]
They're still going through the list. I personally was accepted yesterday, and applied pretty much the same day the application was released.

**Reply by darkanubis0100** [reply]
What a horror, the worst thing is that they are accepting accounts that technically don't have repositories. But those of us who work with GitHub even if we have private repositories should have some priority over those who don't have repositories.

**Reply by linktlh** [reply]
Yea well.. at least there's some amount of acceptance. Better than "100 get to try it, next month 5 more people"

**Reply by darkanubis0100** [reply]
> Yea well.. at least there's some amount of acceptance. Better than "100 get to try it, next month 5 more people"

Totally agree with that
